sopel/config/core_section.py

```diff
@@ -642,11 +908,12 @@ def homedir(self):
     If ``True``, Sopel will send information about the triggered exception to
     the sender of the message that caused the error.
 
-    If ``False``, Sopel will only log the error and will appear to fail silently
-    from the triggering IRC user's perspective.
+    If ``False``, Sopel will only log the error and will appear to fail
+    silently from the triggering IRC user's perspective.
```

Comment:
Another 79 vs. 80 wrapping thing. Sigh. Unnecessary, but at least this one doesn't add any extra lines.